canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -480,6 +482,7 @@ PropertiesMain.propTypes = {
 		containerType: PropTypes.string,
 		enableResize: PropTypes.bool,
 		conditionReturnValueHandling: PropTypes.string,
+		subtitle: PropTypes.bool,
```

Comment:
Why is this a boolean property?  Shouldn't this be controlled by uihints?

Reply:
I added this since this new subtitle section is optional, and a lot of existing uihints already have these properties, so this lets the user opt in to showing the header. Should we have a new property added to uihints instead? I think it would be easier for consuming applications to control it through the propertiesConfig like this.